pkg/cluster/internal/kubeadm/config.go

```diff
@@ -202,6 +208,8 @@ apiServer:
 controllerManager:
   extraArgs:
     enable-hostpath-provisioner: "true"
+networking:
+  podSubnet: {{ .PodSubnet }}
```

Comment:
I had issues when using IPv6 addresses, we should quote it
`podSubnet: "{{ .PodSubnet }}"`

Reply:
done